src/frontend/contexts/LocalDiscoveryContext.tsx

```diff
@@ -27,6 +29,9 @@ export type LocalDiscoveryState = {
 
 // Poll wifi state every 2 seconds
 const POLL_WIFI_STATE_INTERVAL_MS = 2000;
+const ZEROCONF_SERVICE_TYPE = 'mapeo';
```

Comment:
nit: rename to `comapeo`?

```suggestion
const ZEROCONF_SERVICE_TYPE = 'comapeo';
```